Sources/Automerge/scalar.swift

```diff
@@ -2,16 +2,27 @@ import enum AutomergeUniffi.ScalarValue
 
 typealias FFIScalar = AutomergeUniffi.ScalarValue
 
+/// A type that represents a primitive Automerge value
 public enum ScalarValue: Equatable, Hashable {
+    /// A byte buffer
     case Bytes([UInt8])
+    /// A string
     case String(String)
+    /// An unsigned integer
     case Uint(UInt64)
+    /// A signed integer
     case Int(Int64)
+    /// A floating point number
     case F64(Double)
+    /// An integer counter
     case Counter(Int64)
+    /// A timestamp
```

Comment:
Might be worth specifying here that the value is milliseconds since the UNIX epoch

Reply:
Good to know - I'll mark that down.